web/src/components/apps/EmbeddedClusterManagement.tsx

```diff
@@ -369,6 +370,29 @@ const EmbeddedClusterManagement = ({
   }, [nodesData?.nodes?.toString(), hasNodesChanged]);
   // #endregion
 
+  // temporary redirect logic - when user has clicked continue during initial install and went back to the cluster management page
+  useEffect(() => {
+    const appNeedsConfiguration = app?.downstream?.pendingVersions?.length > 0;
```

Comment:
wouldn't this redirect even if it's not the initial install workflow? i can have many deployed versions and some pending versions, which would then redirect every time i visit the cluster management page later after the initial install, right?

Reply:
@sgalsaleh ah you're right. Updated to have a check if it's `fromLicenseFlow`